lib/actions-oclif/local/configure.ts

```diff
@@ -119,9 +119,7 @@ export default class LocalConfigureCmd extends Command {
 	readonly CONNECTIONS_FOLDER = '/system-connections';
 
 	getConfigurationSchema(connectionFileName?: string) {
-		if (connectionFileName == null) {
-			connectionFileName = 'resin-wifi';
-		}
+		connectionFileName ??= 'resin-wifi';
```

Comment:
I switched to this pattern for coffeescript style defaults in a bunch of places, but when we're fully in typescript/know null cannot be passed then it can be changed to use
```ts
	getConfigurationSchema(connectionFileName = 'resin-wifi') {
```
instead